codecov.yml

```diff
@@ -0,0 +1,2 @@
+ignore:
+  - "trollsift/tests"
```

Comment:
@djhoese I have added this change. This excludes the tests from the coverage calculation. My changes only introduced a marginal decrease in coverage here, but to me it does not make sense to track coverage for test files to begin with. Let me know whether you agree.

Reply:
I do not agree. Coverage of tests should almost be 100% every time. It is a good indicator of something being misconfigured if the tests don't have a high coverage percentage.

Additionally, this configuration file is only for `codecov` and not coverage in general (which would be a different config file, `.coveragerc` I think). Anyway, I'd prefer you remove the ignore.

Reply:
I'll remove it then 👍